src/Uno.UWP/System/Launcher.cs

```diff
@@ -1,105 +1,67 @@
-﻿#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
-
-using System;
-using System.Threading.Tasks;
+﻿using System;
 using Uno.Extensions;
-using Uno.Logging;
 using Windows.Foundation;
-using Microsoft.Extensions.Logging;
-using System.Linq;
 using Windows.UI.Core;
 
-#if __ANDROID__
-using Android.Content;
-using Android.Content.PM;
-#endif
-#if __IOS__
-using UIKit;
-using AppleUrl = global::Foundation.NSUrl;
-#endif
-
 namespace Windows.System
 {
 	public static partial class Launcher
 	{
-		public static async Task<bool> LaunchUriAsync(Uri uri)
+		private const string MicrosoftUriPrefix = "ms-";
+
+		private static bool IsSpecialUri(Uri uri) => uri.Scheme.StartsWith(MicrosoftUriPrefix, StringComparison.InvariantCultureIgnoreCase);
```

Comment:
I'm unsure that an uri beginning with `ms-` should be considered as a _Microsoft Uri_. It's a stretch, but you should fully qualify the scheme.

Reply:
The reasoning here is that all the special URIs in UWP start with `ms-`, so this is just the very "raw" filter. In the implementation I then check if the URI is actually supported (by comparing to the known URIs, currently `ms-settings` and if not, the handling proceeds in normal manner, so it should not cause any issues